common/app/common/configuration.scala

```diff
@@ -21,11 +24,42 @@ class GuardianConfiguration extends Logging {
   case class OAuthCredentials(oauthClientId: String, oauthSecret: String, oauthCallback: String)
   case class OAuthCredentialsWithMultipleCallbacks(oauthClientId: String, oauthSecret: String, authorizedOauthCallbacks: List[String])
 
-  protected val configuration = ConfigurationFactory.getNonLoggingConfiguration("frontend", "env")
+  import com.gu.cm.{Configuration => CM}
+  import com.typesafe.config.Config
+  val configuration = {
 
-  private implicit class OptionalString2MandatoryString(conf: com.gu.conf.Configuration) {
-    def getMandatoryStringProperty(property: String) = configuration.getStringProperty(property)
+    val cmLogger = new Logger {
+      override def warn(message: => String): Unit = log.warn(message)
+      override def error(message: => String): Unit = log.error(message)
+      override def error(message: => String, exception: => Throwable): Unit = log.error(message, exception)
+      override def info(message: => String): Unit = log.info(message)
+    }
+
+    val stage = new CM(List(FileConfigurationSource(s"/etc/gu/install_vars")), cmLogger).load.getStringProperty("STAGE").getOrElse("DEV")
```

Comment:
I recommend using `Identity.whoAmI` here as it will automatically use our convention to discover where is the machine running. Provided the instance is tagged correctly, it will populate all the fields of the `Identity` case class.